### DISCUSSION TITLE
Sharing recipes between groups in a single instance

### DISCUSSION THREAD

**Discussion by hanserasmus** [mealie-recipes/mealie#2456] [Feature Request]
I love this software. I am writing this feature request here, to see if other people might have the same use case.

I want to create groups within Mealie, which can be seen as different "families". So main group is everyone under my roof. Second group are my in-laws, third is my best friend etc. The idea is, all my recipes are available to all other groups for them to put in their meal planner, unless I make the recipe private. And the same goes for the other way around. They can create a recipe, and unless they make it private, I will be able to view the recipe and add it to my own meal-planner. This way we can have a collective recipe system which has more diversity.

I don't want to add everyone to my group, because then they cannot do their own meal planning.

Am I the only one who thinks this is a good idea?

**Comment by michael-genson** [comment]
See the households feature here: https://github.com/mealie-recipes/mealie/discussions/2134
A PR is open to partially implement this, with more coming later
